web/components/application/hacker/pages/PageThree/PageThreeConstraints.js

```diff
@@ -0,0 +1,17 @@
+import { atLeastOneCharacter } from '../../../constraints/utils';
+
+export const constraints = {
+  source: atLeastOneCharacter,
+  isCheckedPrivacyPolicy: {
```

Comment:
You have to change the name of these variables as well - right now the submit button is not being enabled even when you fill everything out

Reply:
Right, my bad I didn't check that. Good catch